.github/workflows/integration_tests.yml

```diff
@@ -315,12 +315,12 @@ jobs:
           path: external/vcpkg/installed
           key: dev-vcpkg-${{ env.VCPKG_TRIPLET }}-${{ hashFiles(format('{0}', env.VCPKG_RESPONSE_FILE)) }}-${{ hashFiles('.git/modules/external/vcpkg/HEAD') }}
       - name: Fetch prebuilt packaged SDK from previous run
-        uses: dawidd6/action-download-artifact@v2
+        uses: actions/download-artifact@v4
         if: ${{ github.event.inputs.test_packaged_sdk != '' }}
         with:
           name: 'firebase_cpp_sdk.zip'
-          workflow: 'cpp-packaging.yml'
```

Comment:
Will this correctly pull the packaged SDK from that workflow run? It doesn't try to read from the integration test workflows?

Reply:
I believe the run-id is the important part, since that should be unique. The Unity test workflow uses it to get the build artifacts, and that worked as expected.